examples/examples.go

```diff
@@ -40,6 +40,7 @@ var (
 	//go:embed templates/kubernetes-devcontainer
 	//go:embed templates/nomad-docker
 	//go:embed templates/scratch
+	//go:embed templates/tasks-docker
```

Comment:
We currently embed the starter template in the binary. This is where we add the tasks-docker template to the list (the examples/templates/tasks-docker directory contains its contents copied from https://github.com/coder/registry).